cv-corpus/scs/23.0-2025-09-17/final/en/nhi.md

```diff
@@ -1,7 +1,13 @@
-# *Mehicano* &mdash; Western Sierra Puebla Nahuatl (Zacatlán-Ahuacatlán-Tepetzintla Nahuatl) (`nhi`)
+# *[Western Sierra Puebla Nahuatl]* &mdash; Western Sierra Puebla Nahuatl (`nhi`)
+
+This datasheet is for version 23.0 of the the Mozilla Common Voice *Scripted Speech* dataset 
+for Western Sierra Puebla Nahuatl (`nhi`). The dataset contains 0.6 hours of recorded
+speech (0.1 hours validated) from 6 speakers.
 
 ## Language
```

Comment:
Mention somewhere if there is variation in the text, e.g. between Tenango and Omitlan. Some prefixes are different or something.

Reply:
in the language description